params/config.go

```diff
@@ -212,7 +237,7 @@ func (c *ChainConfig) IsAllowList(blockTimestamp *big.Int) bool {
 	return utils.IsForked(c.AllowListConfig.Timestamp(), blockTimestamp)
 }
 
-// IsIstanbul returns whether num is either equal to the Istanbul fork block or greater.
+// GetFeeConfig returns the *FeeConfig if it exists.
```

Comment:
update comment to reflect the fact that this returns the default fee config if nil.